### PR TITLE
feat-15-added the slider component

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Slider Component.

Files to Check: 
1. [sliders.html](https://github.com/SinchanaMS/Clay/pull/19/files#diff-d5a0412045dc2dfb34dfdd151a4a9df976147d3f9db1ef05889e45fb13878927)
2. [sliders.css](https://github.com/SinchanaMS/Clay/pull/19/files#diff-8418afe92374fab3d3b50f187f37836341a831587b22945d45b3a2fdd3b7966a)
3. [sliderHandler.js](https://github.com/SinchanaMS/Clay/pull/19/files#diff-f7d9c7d10cf1738147ddc0a4e4ae8add01a40d3b872db15f3180f00e2f1898d2)

Preview:

image

### CODE REVIEW

File: docs/sliders.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                <h1 class="wrapper-heading">Sliders</h1>
+                <hr>
+                <div class="description">
+                    Sliders are often the UI control of choice for letting users select a value or range from a fixed
+                    set of options.
+                    Here's a default slider with a minimum value of 1 and a maximum value of 10. Use class <code
+                        class="class-highlight">slider-container</code> in the parent container and class <code
+                        class="class-highlight">slider</code> in the input tag.
+                    <br>
+                    Optional: Use class <code class="class-highlight">slider-range</code> in the parent container of the
```
Comment (manojsarna) [inline docs/sliders.html]: Suggestion : Dont use br tag , handle it with flex or grid

File: js/sliderHandler.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+var slider = document.querySelector(".slider")
+var sliderValue = document.querySelector(".slider-value")
+sliderValue.innerHTML = slider.value
+
+function getSliderValue(e){
+    sliderValue.innerHTML = e.target.value
+    console.log("called")
+}
```
Comment (manojsarna) [inline js/sliderHandler.js]: Suggestion remove all console.log from the code after correcting the code!

**Review by manojsarna — APPROVED** [review]
Overall code looks good!!

You can merge

File: docs/sliders.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            <div class="component-wrapper sliders-wrapper">
+                <h1 class="wrapper-heading">Sliders</h1>
+                <hr>
+                <div class="description">
+                    Sliders are often the UI control of choice for letting users select a value or range from a fixed
+                    set of options.
+                    Here's a default slider with a minimum value of 1 and a maximum value of 10. Use class <code
+                        class="class-highlight">slider-container</code> in the parent container and class <code
+                        class="class-highlight">slider</code> in the input tag.
+                    <br>
```
Comment (srejitk) [inline docs/sliders.html]: Make use of gaps or bottom margin or padding instead of breaks here

File: js/sliderHandler.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+var slider = document.querySelector(".slider")
+var sliderValue = document.querySelector(".slider-value")
+sliderValue.innerHTML = slider.value
+
+function getSliderValue(e){
+    sliderValue.innerHTML = e.target.value
+    console.log("called")
```
Comment (srejitk) [inline js/sliderHandler.js]: Same suggestion as Manoj. Get rid of logs once tested.

**Review by srejitk — APPROVED** [review]
Code Looks good, just get rid of the logs and you're good to go.

**Review by m26unkwn — APPROVED** [review]
Everything good code is readable.

1. Using Semantic is a good way

**Comment by SinchanaMS** [comment]
Thank you for the reviews. I have made the PR changes, merging to temp-dev.

**Pull request merged**